microsoft/testsuites/performance/storageperf.py

```diff
@@ -213,6 +213,18 @@ def _perf_nfs(
         ):
             raise SkippedException(f"{server_node.os.name} not supported")
 
+        # refer
```

Comment:
refer for what? Please summary for below logic, why rh 8 doesn't support udp mode.

Reply:
update